apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -1136,6 +1136,7 @@ mod tests {
     const EXAMPLE_SCHEMA: &str = include_str!("testdata/schema.graphql");
 
     #[test(tokio::test)]
+    #[ignore = "This test doesn't work with native query planning enabled by default"]
```

Comment:
I do not see how all those tests could fail with the new planner. This should be fixed instead of bulk ignoring tests

Reply:
i could reenable these tests by setting the appropriate configuration in the planner, so i think we're good here.